language-zh.cpp

```diff
@@ -0,0 +1,9201 @@
+S("frames per second", "帧率")
+S("monsters killed: %1", "击杀数：%1")
+S("Drawing %1 (layer %2), F1 for help", "绘制 %1 （层 %2）, F1 - 帮助")
+S("hepta floor", "七边形地板")
+S("hexa floor", "六边形地板")
+S("character", "角色")
+S("ESC for menu/quest", "ESC - 菜单/任务")
+S("vector graphics editor", "矢量图形编辑器")
+S("cheat mode", "作弊模式")
+S("heptagonal game board", "七边形棋盘")
+S("triangular game board", "三角形棋盘")
+S("HyperRogue game board", "HyperRogue棋盘")
+S("first page [Space]", "第一页 [空格]")
+S("Configuration:", "设置：")
+S("video resolution", "显示分辨率")
+S("fullscreen mode", "全屏模式")
+S("animation speed", "动画速度")
+S("dist from hyperboloid ctr", "双曲体中心距离")
+S("scale factor", "缩放参数")
+S("wall display mode", "墙体显示模式")
+S("monster display mode", "怪物显示模式")
+S("cross display mode", "交叉显示模式")
+S("background music volume", "背景音乐音量")
+S("OFF", "关")
+S("ON", "开")
+S("distance between eyes", "双眼间距")
+S("framerate limit", "帧率限制")
+S("joystick mode", "摇杆模式")
+S("automatic", "自动")
+S("manual", "手动")
+S("language", "语言")
+S("EN", "CN")
+S("player character", "玩家角色")
+S("male", "男")
+S("female", "女")
+S("use Shift to decrease and Ctrl to fine tune ", "按Shift减少，Ctrl微调")
+S("(e.g. Shift+Ctrl+Z)", " （如Shift+Ctrl+Z）")
+S("the second page [Space]", "第二页 [空格]")
+S("special features [Space]", "特性 [空格]")
+S("see the help screen", "检视帮助")
+S("save the current config", "保存当前设置")
+S("(v) config", "(v) 设置.")
+S("Screenshot saved to %1", "截图保存至%1")
+S("You need an Orb of Teleport to teleport.", "需要拥有传送法球才能传送。")
+S("Use arrow keys to choose the teleport location.", "使用方向键选择传送目标。")
+S("openGL mode enabled", "OpenGL模式开启")
+S("openGL mode disabled", "OpenGL模式关闭")
+S("openGL & antialiasing mode", "OpenGL与抗锯齿模式")
+S("anti-aliasing enabled", "抗锯齿开启")
+S("anti-aliasing disabled", "抗锯齿关闭")
+S("You activate your demonic powers!", "你启动你的恶魔之力！")
+
+// Steam achievement messages
+// --------------------------
+
+S("New Achievement:", "新成就：")
+S("Your total treasure has been recorded in the Steam Leaderboards.", "你的总宝藏数已记录至Steam排行榜。")
+S("Congratulations!", "恭喜！")
+S("You have improved your total high score and %1 specific high scores!", "你更新了你的总最高分及%1个最高分！")
+S("You have improved your total and '%1' high score!", "你更新了你的总最高分及“%1”最高分！")
+S("You have improved your total high score on Steam. Congratulations!", "你更新了你在Steam的总最高分！恭喜！")
+S("You have improved %1 of your specific high scores!", "你更新了你的%1个最高分！")
+S("You have improved your '%1' high score on Steam!", "你更新了你在Steam的“%1”最高分！恭喜！")
+S("You have collected 10 treasures of each type.", "你收集了每种宝藏各10个。")
+S("This is your first victory!", "这是你的首次胜利！")
+S("This has been recorded in the Steam Leaderboards.", "已记录至Steam排行榜。")
+S("The faster you get here, the better you are!", "来得越快，你的水平就越高！")
+S("You have improved both your real time and turn count. Congratulations!", "你更新了你的实时及回合数记录。恭喜！")
+S("You have used less real time than ever before. Congratulations!", "你更新了最短实际时间的记录。恭喜！")
+S("You have used less turns than ever before. Congratulations!", "你更新了最短回合数的记录。恭喜！")

```

Comment:
@zenorogue "less" should be "fewer" here (and update all language files to match)